rlp/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rlp"
-version = "0.4.2"
+version = "0.5.0"
```

Comment:
sorry, didn't spot this, but no need for a bump

Reply:
damn, never do this late.... nice catch